python/pyspark/sql/tests/test_udf_profiler.py

```diff
@@ -333,6 +333,69 @@ def filter_func(iterator):
 
         self.assertEqual(0, len(self.profile_results), str(self.profile_results.keys()))
 
+    @unittest.skipIf(
+        not have_pandas or not have_pyarrow,
+        cast(str, pandas_requirement_message or pyarrow_requirement_message),
+    )
+    def test_perf_profiler_group_apply_in_pandas(self):
```

Comment:
Shall we have a test for `applyInArrow` as well?

Reply:
Good catch!